Update version for current images

ISO50 opened the pull request:
Update to new node 10.x version

ISO50 committed:
Update version for current images

ISO50 commented:
👍

ISO50 commented:
Not sure if this is enough. The documentation is short ;)
If something is missing, please let me know.

ISO50 commented:
@kgalli What is the status of this PR and of this repository in general?

If support is needed please let me know.

kgalli commented:
@ISO50 please update to latest node. I tend to update `current` and `lts` to the latest.

kgalli commented:
@ISO50 please also have a look at the official repo. Seems there are changes we have to move over:
https://github.com/nodejs/docker-node/blob/3e179a85703a6688a26486729b4466a92e818a84/10/jessie/Dockerfile#L17

ISO50 committed:
Update version for current images

- LTE 8.11.3
- Current 10.7.0
- Updated gpg release keys
- Updated Readme and Maintainers

ISO50 commented:
@kgalli 
I've updated Current and LTS and added the the new GPG key.

kgalli commented on .gitignore:
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .idea/
+.vscode
```
Not a big fan of having editor/IDE configs in `.gitignore`. I prefer global setting (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). But due to the fact we already have `.idea` I can not complain :-) ... Just keep in mind that eventually those lines might be removed from `.gitignore`.

kgalli commented:
@ISO50 sorry for the late response. Thanks for the update.

kgalli commented:
👍

ISO50 commented:
👍
